CONTRIBUTING.md

```diff
@@ -46,6 +46,25 @@ Make sure you have the following dependencies installed first:
    mage -v
    ```
 
+## Setting up a go workspace
+
+Setting up go workspace can be helpful when making across modules like `grafana-aws-sdk` and `sqlds` and wanting to see those changes in the Athena data source.
+
+From https://go.dev/blog/get-familiar-with-workspaces:
+> Workspaces in Go 1.18 let you work on multiple modules simultaneously without having to edit go.mod files for each module. Each module within a workspace is treated as a main module when resolving dependencies.
+> 
+> Previously, to add a feature to one module and use it in another module, you needed to either publish the changes to the first module, or edit the go.mod file of the dependent module with a replace directive for your local, unpublished module changes. In order to publish without errors, you had to remove the replace directive from the dependent module’s go.mod file after you published the local changes to the first module.
+
+1. Make a new directory somewhere, for example `athena_workspace`
+2. `cd athena_workspace`
+3. `git clone https://github.com/grafana/athena-datasource.git`
+4. `git clone https://github.com/grafana/grafana-aws-sdk`
+5. `git clone https://github.com/grafana/sqlds`
+6. `go work init ./athena-datasource ./grafana-aws-sdk ./sqlds`
+7. Make modifications in any of these directories and build the backend in `athena-datasource` with `mage` as usual. The changes in these directories will be taken into account.
+
+If you build Grafana locally, you can for example symlink `athena-datasource` to your clone of `github.com/grafana/grafana`'s `data/plugins` directory, e.g. `cd <path to your Grafana repo>/github.com/grafana/grafana/data/plugins && ln -s <path to your workspaces>/athena_workspace/athena-datasource athena-datasource`
```

Comment:
Is this intended as a separate method or part of setting up the workspace? It isn't clear

Reply:
This is how to install this data source in a build of Grafana, but it is not the only way to do so. 

It's information that I thought would be helpful to specify that the `athena-datasource` repo within the go workspace can be continued to be treated as a "plugin" as we usually did before.